src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -534,7 +538,13 @@ const WorkflowView = _.flow(
 
     const inputsValid = _.isEmpty(errors.inputs)
     const outputsValid = _.isEmpty(errors.outputs)
-    return div({ style: { position: 'relative', backgroundColor: 'white', borderBottom: `2px solid ${colors.accent()}` } }, [
+    return div({
+      style: {
+        position: 'relative',
+        backgroundColor: 'white', borderBottom: `2px solid ${terraSpecial()}`,
+        boxShadow: '0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16)'
```

Comment:
The buttons separate visually from the border on the outputs tab due to the extra content there. It's usable, but a little surprising since it breaks the 'tab panel' metaphor.

Reply:
Yeah, I brought this up to Lou and he was ok with it.